drivers/pds/lib/utils.go

```diff
@@ -229,8 +234,28 @@ func GetAndExpectBoolEnvVar(varName string) (bool, error) {
 	return varBoolValue, err
 }
 
+func GetDeploymentTargetID(clusterID, tenantID string) (string, error) {
+	log.Info("Get the Target cluster details")
+	targetClusters, err := components.DeploymentTarget.ListDeploymentTargetsBelongsToTenant(tenantID)
+	if err != nil {
+		log.Errorf("Error while listing deployments %v", err)
```

Comment:
do not log the error here, it should logged in the test

Reply:
done